src/components/menu/menu.ts

```diff
@@ -35,6 +48,35 @@ export class MdMenu {
     this._showClickCatcher = bool;
   }
 
+  /**
+   * This method takes classes set on the host md-menu element and applies them on the
+   * menu template that displays in the overlay container.  Otherwise, it's difficult
+   * to style the containing menu from outside the component.
+   * @param classes: list of class names
+   */
+  private _mirrorHostClasses(classes: string): void {
```

Comment:
Now that I'm changing this over to use @Input, it's a setter anyhow. Hopefully the comment will be sufficient?